metecho/api/views.py

```diff
@@ -270,7 +270,8 @@ def collaborators(self, request, pk=None):
 class TaskViewSet(RepoPushPermissionMixin, CreatePrMixin, ModelViewSet):
     permission_classes = (IsAuthenticated,)
     serializer_class = TaskSerializer
-    queryset = Task.objects.active()
+    queryset = Task.objects.select_related("epic", "epic__project").active()
+    pagination_class = CustomPaginator
```

Comment:
@oluoluoxenfree Okay, now that tasks are paginated you'll need to update how they're stored in the Redux store. Previously we've been storing them as a list of objects, nested under the key of the the parent `epic.id`.

```
{
  [epic.id]: [{ task1... }, { task2 }, ...],
}
```

Now I think we'll want to expand that, similar to how we store Projects.

**Actually, this feels a bit like scope creep.** Let's comment out this line for now, and deal with pagination in a new PR.

```suggestion
    # pagination_class = CustomPaginator
```

Reply:
Broken out into a new story: https://trello.com/c/BctHd23k/301-i-want-task-lists-to-be-paginated

Reply:
Pagination disabled in f9f695e8ad82e0f6c7d31cd015eea634147e3574